docs/dev/new_check_howto.md

```diff
@@ -270,15 +270,91 @@ Run only the integration tests for faster iterations:
 tox -e integration
 ```
 
-The check is almost done. Let's add the final touches.
+The check is almost done. Let's add the final touches by adding the integration configurations.
 
-## Final touches
+## Integration configuration
+### Integration Check configuration file
+#### Parameters
 
-### Populate the README
+Parameters in a configuration file follow these rules:
+
+* Placeholders should always follow this format: `<THIS_IS_A_PLACEHOLDER>`according to the documentation [contributing guidelines][16]: 
+* All required parameters are **not** commented by default.
+* All optional parameters are commented by default.
+* If a placeholders has a default value for an integration (like the status endpoint of an integration), it can be used instead of just a canva  placeholder.
+
+#### Parameters documentation
+
+Each parameter in your integration check `conf.yaml.example` configuration file, must have a paragraph in form of YAML comment strings with the following format:
```

Comment:
I wouldn't use the term "paragraph" since the content is YAML. Maybe "must have a special comment block with the following format:"?
(also note the comments have to be the special extra '#' char)